src/ballet/sbpf/fd_sbpf_loader.c

```diff
@@ -472,18 +474,21 @@ fd_sbpf_load_shdrs( fd_sbpf_elf_info_t *  info,
   return 0;
 }
 
-static int
```

Comment:
This fn was static int, but the wrapped in a fn returning pointer. And not used anywhere else. So I figured to simplify and just keep the pointer one.